<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetItemView</class>
 <widget class="QDialog" name="SetItemView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox{
	border-style:solid;
	border:1px solid black;
}
QPushButton{
	border-style:solid;
	border-radius:3px;
    font-size:14px;
}
QPushButton:focus{
	color:#4aabe9;
    background-color:#f6f7fc;
}
</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>127</x>
     <y>0</y>
     <width>311</width>
     <height>211</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="page"/>
   <widget class="QWidget" name="page_3"/>
   <widget class="QWidget" name="page_4">
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size:14px;</string>
     </property>
     <property name="text">
      <string>本机实验台ID：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size:16px;</string>
     </property>
     <property name="text">
      <string>设置WebIP:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="web_ip">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>150</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style:solid;
border:1px solid black;
font-size:18px;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="studentID">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>150</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style:solid;
border:1px solid black;
font-size:18px;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size:14px;</string>
     </property>
     <property name="text">
      <string>教师机名</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="trachIP">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>80</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style:solid;
border:1px solid black;
font-size:18px;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_5">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>261</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>系统串口列表</string>
     </property>
     <widget class="QComboBox" name="comBoxComPort">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>171</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size:18px;
border-style:solid;
border:1px solid black;</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2"/>
  </widget>
  <widget class="QPushButton" name="okBtn">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>250</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:solid;
background-color:#4aabe9;
color:white;</string>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>250</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:solid;
background-color:#4aabe9;
color:white;</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>111</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:none;
border-right:1px solid #ebedee</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="btn3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>系统设置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>选择串口</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
